twindle-cli/src/renderer/main/components/Tweet/ImageLink.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  /**
+   * @type {Partial<LinkWithImage> | undefined}
+   */
+  export let linkWithImage = undefined;
```

Comment:
`undefined` can be removed, as not specifying anything will make it undefined by default, and the type can be simplified